rotkehlchen/globaldb/cache.py

```diff
@@ -92,7 +109,7 @@ def globaldb_get_general_cache_last_queried_ts(
         value: str,
 ) -> Optional[Timestamp]:
     """Function to get timestamp at which pair key - value was queried last time."""
-    cache_key = compute_cache_key(key_parts)
+    cache_key, _ = compute_cache_key(key_parts)
     cursor.execute(
```

Comment:
This function doesn't make much sense since it is no longer the general cache but the cache needed for that CacheType. I see that this function is only used in tests. Can you maybe move it to a function in `tests/utils/globaldb.py`?